cooltools/api/expected.py

```diff
@@ -1017,25 +1019,43 @@ def expected_cis(
 
     # additional smoothing and aggregating options would add columns only, not replace them
     if smooth:
+        if clr_weight_name is None:
+            # result["count.avg"] = result["count.sum"] / result["n_valid"]
+            cols = {
+                "dist": "dist",
+                "n_pixels": _NUM_VALID,
+                "n_contacts": "count.sum",
+                "contact_freq": "count.avg",
+                "smooth_suffix": ".smoothed",
+            }
+        else:
+            cols = expected_smoothing.DEFAULT_CVD_COLS
+
         result_smooth = expected_smoothing.agg_smooth_cvd(
-            result,
-            sigma_log10=smooth_sigma,
+            result, sigma_log10=smooth_sigma, cols=cols
         )
         # add smoothed columns to the result (only balanced for now)
         result = result.merge(
-            result_smooth[["balanced.avg.smoothed", _DIST]],
+            result_smooth[[cols["contact_freq"] + cols["smooth_suffix"], _DIST]],
```

Comment:
I find this notation very hard to read. Is it possible to retain something like this? 
`"count.sum" + suffix` / without dictionary, having the regular notation

I endorse both readability and flexibility of the code, and here it looks like the balance is shifted towards flexibility.